xga/exceptions.py

```diff
@@ -104,6 +104,46 @@ def __str__(self):
             return 'eSASSNotFoundError has been raised'
 
 
+class CIAONotFoundError(Exception):
+    def __init__(self, *args):
+        """
+        Exception raised if the Chandra Scientific Analysis Software CIAO can not be found on the system.
```

Comment:
'Scientific Analysis Software' is the XMM SAS software name - don't need it here